tests/extension/commands/index.test.ts

```diff
@@ -60,3 +67,13 @@ test('runCLICommand if CLI is installed', async () => {
   expect(terminal.show).toBeCalledTimes(1)
   expect(terminal.sendText).toBeCalledWith('runme run foobar --chdir="/foo"')
 })
+
+test('open markdown as runme notebook', (file: NotebookDocument) => {
+  openAsRunmeNotebook(file)
+  expect(window.showNotebookDocument).toBeCalledWith(file, { viewColumn: 2})
```

Comment:
To assert just extend the mock with arbitrary values.

Reply:
Do I extend in the __mocks__ or direct from the expect with extend? ex) `expect.extend...`

I am getting this as my error
<img width="955" alt="Screen Shot 2022-11-15 at 2 45 26 PM" src="https://user-images.githubusercontent.com/63378463/202040717-7c784356-0a57-4602-86d9-31a14d0c3334.png">

Reply:
this doesnt seem to work
<img width="955" alt="Screen Shot 2022-11-15 at 2 45 26 PM" src="https://user-images.githubusercontent.com/63378463/202041127-ea81bc43-8478-4b89-aa03-8485e84e8670.png">

Reply:
You have to mock `ViewColumn.Beside` which means to create and export an enum in https://github.com/stateful/vscode-runme/blob/main/__mocks__/vscode.ts, then in your test do:

```ts
import { ViewColumn } from 'vscode'
...
expect(window.showNotebookDocument).toBeCalledWith(file, { viewColumn: ViewColumn.Beside })
```